File: JAVA/58. Length of Last Word.java
class Solution {
    public int lengthOfLastWord(String s) {
        char[] ans = s.toCharArray();
        int count=0;
        boolean flag = false;
        for(int i=ans.length-1; i>=0; i--)
        {
            
            if(ans[i]!=' '){
                count++;
                flag=true;
            }
            else {
                if(flag==true)
                    break;
            }
        }
        return count;
    }
}

File: C++/58. Length of Last Word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int n=s.length();
        int count=0;

        for(int i=n-1; i>=0; i--){
            if(s[i]  !=' '){
                count++;
            }
            else if(count > 0){
                break;
            }
        }
        return count;;
    }
};
